rollup-plugins/env-based-exclusion.js

```diff
@@ -0,0 +1,29 @@
+const MagicString = require('magic-string');
```

Comment:
Should we collapse this into the plugin that reed wrote so its just one called "production-pruning" or something. Then i would just need to add
```
if (node.type === 'IfStatement' && node.test.value === '__DEV__') {
            if (node.alternate) {
              s.overwrite(node.start, node.alternate.start, '');
            } else {
              s.overwrite(node.start, node.end, '');
            }
          }
```
Into what @crymall wrote